django/contrib/admin/helpers.py

```diff
@@ -433,9 +441,16 @@ def inline_formset_data(self):
     def forms(self):
         return self.formset.forms
 
-    @property
     def non_form_errors(self):
-        return self.formset.non_form_errors
+        return self.formset.non_form_errors()
```

Comment:
`InlineAdminFormSet.non_form_errors` is undocumented. Nevertheless, I'd add a short release note in breaking change, e.g.
```
* The undocumented ``InlineAdminFormSet.non_form_errors`` property is replaced
  by the ``non_form_errors()`` method. This is consistent with ``BaseFormSet``.
```